src/utils/GitHubType.jl

```diff
@@ -79,7 +79,7 @@ end
 # dictionary into the type `G` with the expectation that the fieldnames of
 # `G` are keys of `data`, and the corresponding values can be converted to the
 # given field types.
-@generated function json2github{G<:GitHubType}(::Type{G}, data::Dict)
+@generated function json2github(::Type{G}, data::Dict) where {G <: GitHubType}
```

Comment:
I should clean JSON2.jl up and do a release; it has all the generated functions to do this parsing automatically.